rotkehlchen/data_import/importers/shapeshift_trades.py

```diff
@@ -111,21 +105,19 @@ def _import_csv(self, write_cursor: DBCursor, filepath: Path, **kwargs: Any) ->
         - InputError if one of the rows is malformed
         """
         with open(filepath, encoding='utf-8-sig') as csvfile:
-            data = csv.DictReader(csvfile)
-            for row in data:
+            for index, row in enumerate(csv.DictReader(csvfile), start=1):
                 try:
+                    self.total_entries += 1
                     self._consume_shapeshift_trade(write_cursor, row, **kwargs)
+                    self.imported_entries += 1
                 except UnknownAsset as e:
-                    self.db.msg_aggregator.add_warning(
-                        f'During ShapeShift CSV import found action with unknown '
-                        f'asset {e.identifier}. Ignoring entry',
-                    )
+                    self.log_warning(index, row, f'Unknown asset {e.identifier}.')
                     continue
                 except DeserializationError as e:
-                    self.db.msg_aggregator.add_warning(
-                        f'Deserialization error during ShapeShift CSV import. '
-                        f'{e!s}. Ignoring entry',
-                    )
+                    self.log_warning(index, row, f'Deserialization error: {e!s}.')
+                    continue
+                except SkippedCSVEntry as e:
+                    self.log_debug(index, row, str(e))
```

Comment:
So yeah as I said before. If the user should not care or need to see something do not send it via websocket. I am not sure here. What is this? Should the user see it? Is it information he can act on?

Reply:
Since we are sending the number of total and successfully imported rows to the frontend for the user to see, shouldn't we try to give some type of explanatory message for every row that is skipped? So it doesn't say something like "Imported 10/15 rows" and then only show 3 messages about rows that had problems and leave the user wondering what happened on the other skipped rows?

You can see the messages being logged here by looking at where the `SkippedCSVEntry` exception is raised on lines 73, 75, and 83 [here](https://github.com/rotki/rotki/blob/21ca43e13af1e84248ab3f32f4e74ee32abe6c50/rotkehlchen/data_import/importers/shapeshift_trades.py#L73). Actually, in this particular case, these should maybe be changed to `UnsupportedCSVEntry` so they get logged as warnings?